tests/unit/test_correction.py

```diff
@@ -115,3 +116,30 @@ def test_correction_schema_coa():
     print(errors)
 
     assert is_valid
+
+
+def test_correction_schema_cp_special_resolution():
```

Comment:
If you made it a component, then the test would be more like https://github.com/bcgov/business-schemas/blob/73b6614919d1ea618ce5204b49ef7411cd0f2d41/tests/unit/test_alteration.py#L37

Yes, not all are like that, but that's a reviewer issue as that is our preferred direction.

Reply:
fixed, used common header